README.md

```diff
@@ -145,13 +146,28 @@ Facilitators:
 - Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Cisco
 - Zach Corleissen ([@zacharysarah](https://github.com/zacharysarah)), Linux Foundation
 
-### GitHub Management
```

Comment:
IMO github mgmgt is now the community infrastructure team, so I'm excited to see this starting up for real!